src/platform/forms-system/test/js/helpers.unit.spec.js

```diff
@@ -430,8 +430,9 @@ describe('Schemaform helpers:', () => {
           },
         },
       };
-
-      const output = JSON.parse(transformForSubmit(formConfig, formData));
+      const transformed = transformForSubmit(formConfig, formData);
+      const output =
+        typeof transformed === 'object' ? transformed : JSON.parse(transformed);
```

Comment:
I don't see a problem with this change, but I was just curious about why it's included in this PR. Does this have anything to do with the 403 and 404 tests?